pretty_benchmark_result.ts

```diff
@@ -32,20 +32,20 @@ export function prettyBenchmarkResult(
 
 function _prettyBenchmarkResultCb(
   results: BenchmarkRunResult,
-  options: prettyBenchmarkResultOptions,
+  options?: prettyBenchmarkResultOptions,
 ) {
-  if (options.nocolor) c.setColorEnabled(false);
+  if (options && options.nocolor) c.setColorEnabled(false);
```

Comment:
With the pattern above, options will always be an object, so you can just check options.nocolor.

If you use the overload, then this would be `opts.nocolor`.

You can also do the new operator which TS supports and with none of the changes above, would become. 

`if (options?.nocolor) c.setColorEnabled(false);`

Reply:
Thanks, dont know why I didnt use the `?`

Reply:
=)